packages/graphql-transformers-e2e-tests/src/__tests__/KeyTransformer.e2e.test.ts

```diff
@@ -229,6 +230,36 @@ test('Test query with three part secondary key.', async () => {
     expect(items.errors.length).toBeGreaterThan(0);
 })
 
+test('Test query with three part secondary key, where sort key is an enum.', async () => {
+    const hashKey = '2018-06-01T00:01:01.000Z';
+    const sortKey = 'UNKNOWN';
+    await createItem('order1', sortKey, 'list1', '2018-01-01T00:01:01.000Z');
+    await createItem('order2', sortKey, 'list2', hashKey);
+    await createItem('order3', sortKey, 'item3', '2018-09-01T00:01:01.000Z');
+    let items = await itemsByCreatedAt(undefined);
+    expect(items.data).toBeNull();
+    expect(items.errors.length).toBeGreaterThan(0);
+    items = await itemsByCreatedAt(hashKey);
+    expect(items.data.itemsByCreatedAt.items).toHaveLength(1)
+    items = await itemsByCreatedAt(hashKey, { beginsWith: sortKey });
+    expect(items.data.itemsByCreatedAt.items).toHaveLength(1)
+    items = await itemsByCreatedAt(hashKey, { eq: sortKey });
+    expect(items.data.itemsByCreatedAt.items).toHaveLength(1)
+    items = await itemsByCreatedAt(hashKey, { between: [sortKey, sortKey] });
+    expect(items.data.itemsByCreatedAt.items).toHaveLength(1)
+    items = await itemsByCreatedAt(hashKey, { gt: '2018-08-01' });
```

Comment:
Why comparing the STATUS with a date string?

Reply:
Thanks for catching it, that was a late night typo.